### ISSUE TITLE
get rid of twisted.web.http.Request.stack

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6956]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#6956|
|Type|enhancement|
|Created|2014-02-09 14:22:05Z|

It is undocumented, untested, and unused (not to mention useless).

Attachments:

* [6956-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/113/113212e51e8b2950748f7305c7ad9dad7104b605/dd765ddb92a6a60ac2c3365eee8ebe055ea68a05.diff) (472 bytes) - added by adiroiban on 2014-02-09 16:21:51Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6956 6956
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1391955725000000 1391955725000000
changetime__1391964840000000 1391964840000000
version__None None
owner__exarkun exarkun
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban commented|
|-|-|

@exarkun will you take care of that?

If not, I can work on this.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

I wasn't planning to work on it.  I can review a patch, though. :)

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @exarkun|
|-|-|

I have attached a patch.

It just removes the variable.

I have written an empty topfile... since in a previous review I got feedback that for simple changes the topfile should be emtpy.


Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thanks.  Looks good to me.  (If you could generate your git patches without the a/ b/ prefixes though that'd be great. :)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [41617]) Apply 6956-1.diff - remove unused `Request.stack`

Author: adiroiban
Reviewer: exarkun
Fixes: [#6956](https://github.com/twisted/twisted/issues/6956)

`twisted.web.http.Request` had a `stack` attribute for a long time, set partway through
request processing.  Nothing ever used this attribute and it isn't needed for anything.
So, remove it.
